src/book/03-guides/04-general/04-documentation.mdx

```diff
@@ -0,0 +1,82 @@
+---
+section: Guides
+chapter: General
+title: Documentation
+description: How to properly document code.
+slug: /guides/general/documentation
+---
+
+## Conventions
+
+We use tokens called "directives" or "tags" to annotate different parts of a comment with rich metadata. These directives start with `@` or `\\` (`@` is preferred) and are followed by the directive name, arguments (for some directives), and a single line of description.
```

Comment:
`\\`? I thought it was `\`?

Reply:
Indeed, but don't bother mentioning that they can use that tell them to use @

Reply:
I concur

Reply:
The documentation that I link has `\`